tests/sync/test_llh.py

```diff
@@ -0,0 +1,14 @@
+from utils import get_api_key
+
+import cohere
+from cohere.responses import LogLikelihoods
+
+TEST_MODEL = "command-light"
+API_KEY = get_api_key()
+co = cohere.Client(API_KEY)
+
+
+async def test_basic_llh():
```

Comment:
this is sync tests. I don't think we use async (and we seem to use `unittest.TestCase`, not just functions)